html/static_index.html.haml

```diff
@@ -0,0 +1,76 @@
+!!!
```

Comment:
Couldn't the HTML file be shared between the export and the server-rendered version?